gazebo_ros_control/src/default_robot_hw_sim.cpp

```diff
@@ -306,7 +306,9 @@ void DefaultRobotHWSim::writeSim(ros::Time time, ros::Duration period)
         break;
 
       case POSITION:
-#if GAZEBO_MAJOR_VERSION >= 4
+#if GAZEBO_MAJOR_VERSION >= 9
+        sim_joints_[j]->SetPosition(0, joint_position_command_[j], true);
+#elif GAZEBO_MAJOR_VERSION >= 4
         sim_joints_[j]->SetPosition(0, joint_position_command_[j]);
 #else
         sim_joints_[j]->SetAngle(0, joint_position_command_[j]);
```

Comment:
kinetic requires gazebo7 so we can ditch `SetAngle` and just put the old `SetPosition` call in an `#else` block